DAOS-9989 test: Enable regular weekly tcp testing

phender opened the pull request:
Remove the weekly tcp/ucx test stages from the weekly-testing branch as
they are being moved to their own provider-testing-* branches.

Signed-off-by: Phillip Henderson <phillip.henderson@intel.com>

brianjmurrell reviewed (commented):
[How] is this different than simply reverting fd1499a?  I wonder if actually reverting that would make it more obvious that this is effectively backing that landing out.

Maybe the subject and/or description of the commit could be more clear about that?  I.e. changing _Enable_ in the subject to _Disable_ or _Revert_ or something?

phender commented:
> [How] is this different than simply reverting [fd1499a](https://github.com/daos-stack/daos/commit/fd1499a156295daef3878ce866c86159fd85bc1a)? I wonder if actually reverting that would make it more obvious that this is effectively backing that landing out.
> 
> Maybe the subject and/or description of the commit could be more clear about that? I.e. changing _Enable_ in the subject to _Disable_ or _Revert_ or something?

https://github.com/daos-stack/daos/commit/fd1499a156295daef3878ce866c86159fd85bc1a also made some other minor changes to the normal weekly testing. It:

- removed the Functional on CentOS 7 stage
- added checkboxes to optionally skip stages when using the Build with Parameters page
